torch/csrc/utils/python_dispatch.cpp

```diff
@@ -30,6 +30,7 @@ c10::optional<c10::DispatchKey> parseDispatchKey(const std::string& k) {
     {"cpu", c10::DispatchKey::CPU},
     {"cuda", c10::DispatchKey::CUDA},
     {"xla", c10::DispatchKey::XLA},
+    {"math", c10::DispatchKey::Math},
```

Comment:
Hmm. Maybe we should replicate the casing of these entries, would make the Python tests easier to read. No need to do here. Maybe you can ask a bootcamper to do it.